### ISSUE TITLE
Registration of new combinations and replacement names (nomen novum)

### ISSUE THREAD

**Issue by uqmchuvo** [seq-code/registry#30]
The rank of species and subspecies should include the options to chose the proposal as new combination or replacement name. If those are selected, the page should support the basonym selection from the existing names (validated earlier). Basonym as well as the designation of proposal (comb. nov. or nom. nov.) needs to be reflected in the nomeclature section of the page.

**Comment by lmrodriguezr** [comment]
Depends on #33

**Comment by lmrodriguezr** [comment]
Also, important to note:
> **Rule 27**
> Placement of a species or subspecies epithet into a genus or species that is illegitimate does not preclude the legitimacy of the species or subspecies epithet.

**Comment by uqmchuvo** [comment]
Yes, this is correct (Rule 27), and it is simply saying that the nomenclature status of the basonym is not affected if it is placed into the genus that is later recognised as illegitimate. Note that to be recognised as illegitimate, the genus name needs first to be validly published...New combinations can be proposed only based on validly published basonyms (species name). It doesn't matter what happens later with the new genus itself...

**Comment by uqmchuvo** [comment]
The same is applied to the subspecies...
